articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -141,69 +141,69 @@ The thumbprint of the certificate(s) is also required. A guide on how to find th
 
-   - **Repair Manager** - Repair manager allows you to enable patch orchestration on node types with a durability of bronze, which helps keep your virtual machines up to date.
+   - **Repair Manager** - Select **Yes** to use Repair Manager to enable patch orchestration on node types with a durability of bronze, which helps keep your VMs up to date.
 
    - **Service Fabric deployment package URL** - Specify the URL to download the Service Fabric deployment package from. For off-line scenarios, download the service fabric package from the URL specified and upload it to a blob and enable anonymous access and specify the URL here.
 
    - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
 
-   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group. Note that to create a Service Fabric cluster, the resource group must be empty.
+   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group. Note that to create a Service Fabric cluster, the resource group must contain no other resources.
 
    - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
 
    ![Create Service Fabric cluster > Basics](images/azs-browser-create-sf-basics.png)
 
-5. In the **Network Settings** blade, enter the following information and click **OK**:
+5. In the *Network Settings* blade, enter the following information and click **OK**:
 
-   - **Service Fabric TCP Port** - Service Fabric cluster TCP gateway port to be used to connect using Service Fabric client.
+   - **Service Fabric TCP Port** - Service Fabric cluster TCP gateway port to use to connect using the Service Fabric client.
 
-   - **Service Fabric HTTP Port** - Service Fabric cluster HTTP gateway port to be used to connect using Service Fabric Explorer.
+   - **Service Fabric HTTP Port** - Service Fabric cluster HTTP gateway port to use to connect using the Service Fabric Explorer.
 
    - **Service Fabric Reverse Proxy Port** - Service Fabric cluster reverse proxy port
 
-   - **Custom Application Endpoints** - Custom endpoints allow for connections to applications running on this cluster. Enter endpoints separated by comma: 80,8080,8081.
-
-   - **Custom ports to open in the Network Security Group** - Custom ports to allow for the Network Security Group. Enter ports separated by comma: 3389,80,8080,8081.
+   - **Custom Application Endpoints** - Custom endpoints allow for connections to applications running on this cluster. Enter endpoints separated by commas, for example, 80,8080,8081.
 
+   - **Custom ports to open in the Network Security Group** - Custom ports to allow for the Network Security Group. Enter ports separated by comma, for example, 3389,80,8080,8081.

```

Comment:
I think the phrasing of the two paragraphs above need to be consistent with each other, but not sure which one is correct...

And don't forget the s on the end of commas in the second paragraph.

Reply:
Have modified the wording slightly, I think it's better?